Sagemaker docs (#182)
iterative/mlem.ai: content/docs/extensions/deployment/sagemaker.md

## Description

> MLEM SageMaker allow you to deploy MLEM models to AWS SageMaker. You can learn
> more about SageMaker
> [here](https://docs.aws.amazon.com/sagemaker/latest/dg/whatis.html).

Reviewer: This should be the intro sentence TBH (before **Description** or maybe we don't even need a `## Description` header (maybe thus move this one change to #179?).

We can condense it even more too:

```suggestion
This extension allows you to deploy MLEM models to AWS [SageMaker].

[sagemaker]: https://docs.aws.amazon.com/sagemaker/latest/dg/whatis.html
```

One question on terminology though: is "MLEM model" the same as "MLEM model object"? Also sounds a lot like "ML model". We have to think this through so it's clear, I think. Maybe phrase it as `deploy Ml models from MLEM to `

> ### Preparing infrastructre

Reviewer 2: typo., could you please run Grammarly or something on top of the text? (I see other typos, English syntax mistakes, etc)

Reviewer: I use Code Spell Checker on VS Code and it helps fix typos, which is 80% of the problem.

I've just recently started trying Grammarly BTW and I'm not yet sure whether I can recommend it. But it can certainly help improve syntax marginally at least for some people.

Reviewer: Also, I'd call this section
```suggestion
## Prerequisites
```
so it's more general (applicable to all pages)
